### ISSUE TITLE
email

### ISSUE THREAD

**Issue by SongDrop** [shaftoe/curriculum-vitae#3]
It's a really cool site, but people cannot connect you if you don't have email:/

**Comment by shaftoe** [comment]
@SongDrop fair point. There's a whole section with how to get in touch with me though: https://cv.l3x.in/links/
I suppose I should make a "contact me" link more visible, thanks for pointing it out

**Comment by SongDrop** [comment]
What's the easiest way to send you email Sir?

**Comment by shaftoe** [comment]
https://a.l3x.in/contact/ should do
